output/output_tools.py

```diff
@@ -0,0 +1,2 @@
+# This file will be used for storing tools (functions) that aren't related to the listed issues on main branch.

```

Comment:
Tools and functions can be used interchangeably, but they are most often associated with 2 different things in this context. I would consider modifying this to "tools and functions".